gnovm/stdlibs/native.go

```diff
@@ -0,0 +1,537 @@
+// XXX TEST GITHUB ACTION
```

Comment:
This can be removed I think (when I run `go generate` it's no longer here).

Reply:
Yes, didn't push this yet but this is fixed :)